opendebates/tests/test_context_processors.py

```diff
@@ -11,3 +15,16 @@ def test_number_of_votes(self):
             mock_cache.get.return_value = 2
             context = global_vars(mock_request)
             self.assertEqual(2, int(context['NUMBER_OF_VOTES']))
+
+
+class ThemeTests(TestCase):
+
+    def setUp(self):
+        self.idea = SubmissionFactory()
+
+    def test_email_url(self):
+        settings.SITE_THEME['HASHTAG'] = 'TestHashtag'
```

Comment:
Should probably use override_settings here so we don't change the settings permanently during the test.